src/components/__stories__/simple_nav.tsx

```diff
@@ -0,0 +1,23 @@
+import { action, storiesOf } from "@kadira/storybook"
+import * as React from "react"
+
+import styled from "styled-components"
+import SimpleNav from "../simple_nav"
+import TextLink from "../text_link"
+
+import * as fonts from "../../assets/fonts"
+
+const LinkContainer = styled.span`
+  position: relative;
+  top: -8px;
+  font-family: ${ fonts.primary.fontFamily };
+`
+
+storiesOf("Simple Nav", SimpleNav)
+  .add("Simple Nav Bar", () => (
+    <SimpleNav>
+      <LinkContainer>
+        <TextLink href="https://www.artsy.net">Back To Artsy</TextLink>
```

Comment:
I considered abstracting these style rules into the link component. The `top: -8px` is what discouraged me. I thought it was pretty specific to this element. I thought it may make sense to keep the link component lightweight and leave the responsibility of determining how it's styled to the user. 

Maybe there is a happy medium and some things could be passed in as props. Thoughts @l2succes @alloy @yuki24 @broskoski?